Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/BreadCrumbs.cpp

```diff
@@ -220,6 +248,26 @@ namespace AzQtComponents
         QWidget::resizeEvent(event);
     }
 
+    QString BreadCrumbs::generateIconHtml(int index)
+    {
+        QString imagePath;
+
+        if (index >= 0 && index < m_currentPathIcons.size())
+        {
+            imagePath = m_currentPathIcons[index];
+        }
+
+        if (imagePath.isEmpty())
+        {
+            imagePath = m_defaultIcon;
+        }
+
+        return !imagePath.isEmpty() ? QStringLiteral("<img width=\"16\" height=\"16\" style=\"vertical-align: middle\" src=\"%1\">%2%2")
```

Comment:
suggestion: This might actually by a nice use case for a raw string literal (to avoid all the `\"` everywhere 😛)

```
QStringLiteral(R"--(<img width="16" height="16" style="vertical-align: middle" src="%1">%2%2)--")
```